tests/test_bundle.py

```diff
@@ -112,37 +96,70 @@ async def test_build_deploy_knative_charms(ops_test: OpsTest):
     )
 
 
-async def test_cloud_events_player_example(ops_test: OpsTest):
-    await ops_test.run(
-        "kubectl",
-        "apply",
-        "-f",
-        "./examples/cloudevents-player.yaml",
-        check=True,
-    )
-    await ops_test.run(
-        "kubectl",
-        "wait",
-        "--for=condition=ready",
-        "ksvc",
-        "cloudevents-player",
-        "--timeout=5m",
-        check=True,
+RETRY_FOR_MINUTE = Retrying(
+    stop=stop_after_delay(60),
+    wait=wait_fixed(5),
+    reraise=True,
+)
+
+
+@pytest.fixture()
+def wait_for_ksvc():
+    """Waits until KNative Serving Service is available, to a maximum 1 minute"""
+    lightkube_client = Client()
+    for attempt in RETRY_FOR_MINUTE:
+        with attempt:
+            ksvcs = list(lightkube_client.list(KSVC))
+
+
+@pytest.fixture()
+def remove_cloudevents_player_example():
+    """Fixture that attempts to remove the cloudevents-player example after a test has run"""
+    yield
+    lightkube_client = Client()
+    lightkube_client.delete(KSVC, "cloudevents-player")
```

Comment:
You are missing the `namespace` here, otherwise it [won't find it](https://github.com/canonical/knative-operators/actions/runs/4308934326/jobs/7515750777#step:6:436) as it will [look in `default`](https://github.com/canonical/knative-operators/actions/runs/4308934326/jobs/7515750777#step:6:420).

Reply:
yep that's a mistake.  Will fix